app/src/main/java/com/github/scribeWizTeam/scribewiz/models/MusicNoteModel.kt

```diff
@@ -3,43 +3,42 @@ package com.github.scribeWizTeam.scribewiz.models
 import com.google.firebase.firestore.ktx.firestore
 import com.google.firebase.firestore.ktx.toObject
 import com.google.firebase.ktx.Firebase
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.tasks.await
 
 data class MusicNoteModel (
-    override var id: String,
+    override var id: String = "",
     var name: String = "new note",
 ) : Model {
+
     companion object Controller {
         const val COLLECTION = "MusicNotes"
 
-        private const val NOTE_ID = "noteId"
-        private const val NOTE_NAME = "noteName"
-
         fun getAllNotesFromUser(user: UserModel) : Set<MusicNoteModel> {
             val db = Firebase.firestore
 
             val notes : MutableSet<MusicNoteModel> = mutableSetOf()
 
```

Comment:
Great use of runBlocking to manage elegantly synchronisation, I will create a function this sprint for it to be a bit easier to read